packages/driver/src/cy/commands/actions/click.ts

```diff
@@ -117,11 +117,35 @@ export default (Commands, Cypress, cy: $Cy, state, config) => {
         // figure out the options which actually change the behavior of clicks
         deltaOptions = $utils.filterOutOptions(options, defaultOptions)
 
-        options._log = Cypress.log({
-          message: deltaOptions,
-          $el,
-          timeout: options.timeout,
-        })
+        // create a new object with all user-provided options and exclude null values
+        const clickOptions = Object.keys(userOptions)
+        .reduce((obj, key) => {
+          if (options[key] !== null) {
+            obj[key] = options[key]
+          }
+
+          return obj
+        }, {})
+
+        // Add the position property to the clickOptions object if it's not null or undefined
+        if (options.position !== null && options.position !== undefined) {
+          clickOptions.position = options.position
+        }
+
+        // Only log the message if the clickOptions object has any properties
+        if (Object.keys(clickOptions).length > 0) {
+          options._log = Cypress.log({
+            message: [clickOptions],
```

Comment:
I would expect this message to be formatted like the params passed in, so
```js
.click('topRight')
```
would display like:
```js
  - click topRight
```
instead of 
```md
  - click { position: topRight }
```
and if multiple params are passed,
```js
.click(5, 60, { force: true })
```
```md
  - click 5, 6, { force: true }
```